app/src/main/java/org/ole/planet/myplanet/model/RealmMyTeam.java

```diff
@@ -89,6 +89,14 @@ public static void insertMyTeams(String userId, JsonObject doc, Realm mRealm) {
         return ids;
     }
 
+    public static String getTeamCreator(String teamId, Realm realm) {
+        List<RealmMyTeam> teams = realm.where(RealmMyTeam.class).equalTo("teamId", teamId).findAll();
```

Comment:
@ChiragAwale user findFirst() instead of findAll() so you will get single update.